### PR TITLE
image upload

### LINKED ISSUE #39: Setup Amazon S3 or Cloudinary
Allow for uploading of images. Keep our use-cases in mind. We may need to be able to upload multiple images.

\- Sign up to Amazon S3/ Cloudinary and set it up. Feel free to create a new gmail account for the team if you do not want to use your personal account. 
-\*Do not commit api key to repo. Share on Slack if you need to share it across the team\*
\- Create a route for image upload
\- save to storage and retrieve the url to save onto your database where appropriate 

If using Node.js, consider using https://www.npmjs.com/package/multer

### PR DESCRIPTION
**Pull request by gurpartap0306**
### What this PR does (required):
- fixes #39 

### Screenshots / Videos (front-end only):
- Post at least one screenshot to show the feature in action. **Only required in front-end work**

### Any information needed to test this feature (required):
- Post what steps to take to be able to test this feature (eg. "sign up as a new user and upload a profile picture from the navbar")

### Any issues with the current functionality (optional):
- List any problems where you were not able to complete all tasks on ticket
- List any problems this PR may cause for the project or other active PRs
- Post a screenshot/video of the problem, along with detailed console outputs where applicable

### CODE REVIEW

File: server/config/cloudinaryConfig.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        });
+      },
+      {
+        resoulce_type: "auto",
+        folder: folder,
+      }
+    );
+  });
+};
+//module.exports = cloudinaryConfig;
```
Comment (ezeilo-su) [inline server/config/cloudinaryConfig.js:29]: Let's remove unused code

File: server/middleware/multer.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  cloudinary: cloudinary,
+  params: {
+    forder: "DEV",
+  },
+});
+
+const uploader = multer({ storage: storage });
+
+module.exports = uploader;
+*/
```
Comment (ezeilo-su) [inline server/middleware/multer.js:32]: Let's remove unused code

**Review by ezeilo-su — COMMENTED** [review]
Nice one :+1:

File: server/app.js [hunk 3]
```diff
@@ -46,6 +46,34 @@ app.use("/auth", authRouter);
 app.use("/users", userRouter);
 app.use("/profile", profileRouter);
 
+const upload = require("./middleware/multer");
+const cloudinary = require("./config/cloudinaryConfig");
+const fs = require("fs");
+app.use("/upload", upload.array("picture"), async (req, res) => {
```
Comment (bonnieli) [inline server/app.js:52]: you might want to move this into it's own upload route so you don't crowd `app.js`

**Review by bonnieli — APPROVED** [review]
other than my suggested comment, this is good!

**Pull request merged**